package.json

```diff
@@ -71,6 +71,7 @@
     "lint:package-names": "./tooling/lint-package-names.sh",
     "lint:staged": "lint-staged",
     "test": "babel-node ./tooling/index.js test",
+    "test:ci": "babel-node ./tooling/index.js ci --integration",
```

Comment:
What is the different between this and the `test:github` command?

Reply:
`test:github` is used for the github checks
`test:ci` is used for integration testing on github and the file name is `ci.js`

The naming is a bit "i need a quick name" and can be redone, you have any suggestions?

Reply:
Ah, I see. I think `test:ci:integration` and `test:ci:github` feel more readable to me.

Reply:
👍